tools/regalloc/regalloc.ml

```diff
@@ -157,14 +173,41 @@ let collect_in_stats (cfg_with_infos : Cfg_with_infos.t)
     num_high_pressure_points
   }
 
+let safe_pow10 n =
```

Comment:
Consider adding a comment that this is intended to be the same heuristic as in regalloc_utils.

Reply:
Thanks, I meant to have such a note and then forgot about it.